index.js

```diff
@@ -1,7 +1,9 @@
 'use strict'
 
-if (process.env.FFMPEG_BIN) {
-  module.exports = process.env.FFMPEG_BIN
+var pkg = require("./package");
+
+if (process.env.BIN) {
```

Comment:
IMO `$BIN` is too generic, it might clash with other CLI tools in unpredictable ways. I'm in favour of `$FFMPEG_BIN` & `$FFPROBE_BIN`, respectively. 👍 for renaming it though!

Reply:
Noted 👍